src/libstd/path.rs

```diff
@@ -1529,6 +1542,9 @@ impl From<PathBuf> for Arc<Path> {
 
 #[stable(feature = "shared_from_slice2", since = "1.24.0")]
 impl<'a> From<&'a Path> for Arc<Path> {
+    /// Converts a `PathBuf` into a `Arc<Path>`.
+    /// This conversion happens in place.
```

Comment:
I don't think we need to specify in the docs which functions are marked inline, rustdoc already renders the `#[inline]` attribute if I recall correctly.

Reply:
It does indeed so unneeded.

Reply:
So I assume I can delete this comment then and rustdoc is adding an `#[inline]` or should I add it?

Reply:
Actually, I take it back, `#[inline]` is _not_currently rendered by rustdoc. Hmmm

---------

![screen shot 2018-07-09 at 9 17 29 am](https://user-images.githubusercontent.com/416575/42452757-014cc88c-8359-11e8-9a8d-82ca793ac8b5.png)

---------

![screen shot 2018-07-09 at 9 17 35 am](https://user-images.githubusercontent.com/416575/42452761-0333781c-8359-11e8-95ab-fdf166c722ba.png)

---------

https://doc.rust-lang.org/nightly/std/vec/struct.Vec.html#method.new

Reply:
It should be though! I open an issue for it.

Reply:
"in place" sounds different than "this function is marked inline" to me; we should find a better wording if it's not, or remove it if it is supposed to communicate that.